frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/_add-capacity-modal.scss

```diff
@@ -4,6 +4,18 @@
   display: none;
 }
 
+.add-capacity-modal__input input {
+  max-width: 16rem;
```

Comment:
I feel, px should be preferred here as rem depends on root's node size and it can be changed by browser settings.

Reply:
also targeting html tag direct should be avoided as later on, if any input tag gets added, it can break the style.